Realm/RLMRealm_Private.h

```diff
@@ -33,7 +35,7 @@ void RLMRealmTranslateException(NSError **error);
 @interface RLMRealm ()
 
 @property (nonatomic, readonly) BOOL dynamic;
-@property (nonatomic, readwrite) RLMSchema *schema;
+@property (nonatomic, readwrite, null_unspecified) RLMSchema *schema;
```

Comment:
Looks like it was annotated as such due to a [bug](https://github.com/realm/realm-cocoa/commit/d80e58bd262e) @bdash found. I'll change it if the tests pass.